HoloRepositoryDemoApplication/Assets/HoloStorageConnector/Scripts/HoloStorageClient.cs

```diff
@@ -130,16 +133,61 @@ public static IEnumerator GetHologram(Hologram hologram, string HolgramID)
         {
             string GetHologramUri = $"{BaseUri}/holograms/{HolgramID}";
             yield return GetRequest(GetHologramUri);
-            try
+
+            if (WebRequestReturnData != null)
             {
                 JSONNode HologramJson = JSON.Parse(WebRequestReturnData);
-                Hologram Hologram = JsonToHologram(HologramJson);
+                Hologram Hologram = JsonToHologram(HologramJson, HolgramID);
                 CopyProperties(Hologram, hologram);
```

Comment:
I noticed that some variables here have a capitalised name and some don't

Reply:
yes, the capitalised variables  is used to distinguish with the parameter

Reply:
Okay sure, but actually I found that there are a couple of variables in this file in PascalCase. I think according to the [naming conventions](https://github.com/ktaranov/naming-convention/blob/master/C%23%20Coding%20Standards%20and%20Naming%20Conventions.md) it might be better to have all local variables in camelCase.

In this particular case you could e.g. name the argument that holds the result `resultHologram` or you could name the hologram in line 140 `receivedHologram` or `hologramFromJson`.